src/OpenGL/Silk.NET.OpenGL/GL.cs

```diff
@@ -13,26 +18,77 @@ namespace Silk.NET.OpenGL
 {
     public partial class GL
     {
-        [Obsolete
-        (
-            "Parameterless GetApi calls are deprecated and will be removed in a future release. Please create" +
-            "your GL instances using a context"
+        [Obsolete(
+            "Parameterless GetApi calls are deprecated and will be removed in a future release. Please create" + "your GL instances using a context"
         )]
-        public static GL GetApi() => LibraryLoader<GL>.Load
-            (new GLCoreLibraryNameContainer(), SilkManager.Get<GLSymbolLoader>());
-
-        public static GL GetApi(IGLContextSource contextSource) => GetApi
-        (
-            contextSource.GLContext ?? throw new InvalidOperationException
-                ("The given IGLContextSource is not configured with a context.")
-        );
-
-        public static GL GetApi(IGLContext ctx) => GetApi((INativeContext)ctx);
+        public static GL GetApi() => LibraryLoader<GL>.Load(new GLCoreLibraryNameContainer(), SilkManager.Get<GLSymbolLoader>());
```

Comment:
This change goes against the code style for the long line limit of 120 characters. You can solve this using:
```suggestion
        public static GL GetApi()
            => LibraryLoader<GL>.Load(new GLCoreLibraryNameContainer(), SilkManager.Get<GLSymbolLoader>());
```